src/calculateRank.js

```diff
@@ -27,26 +57,37 @@ function calculateRank({
     issues * ISSUES_OFFSET +
     stargazers * STARS_OFFSET +
     prs * PRS_OFFSET +
-    followers * FOLLOWERS_OFFSET
-  ) / totalRepos;
+    followers * FOLLOWERS_OFFSET + 
+    totalRepos * REPO_OFFSET 
+  ) / 100;
+
+  const normalizedScore = normalcdf(score, TOTAL_VALUES, ALL_OFFSETS) * 100;
 
   let level = "";
 
-  if (score == FIRST_STEP) {
-    level = "B";
-  } else if (score > FIRST_STEP && score <= SECOND_STEP) {
-    level = "B+";
-  } else if (score > SECOND_STEP && score <= THIRD_STEP) {
-    level = "A";
-  } else if (score > THIRD_STEP && score <= FOURTH_STEP) {
-    level = "A+";
-  } else if (score > FOURTH_STEP && score <= FIFTH_STEP) {
-    level = "A++";
-  } else if (score > FIFTH_STEP) {
+  if (normalizedScore < RANK_S_VALUE) {
```

Comment:
normalizedScore is for some reason reverse that means the better your score is the normalized  score would be closer to 0 

Users like 
- yyx990803
- kentcdodds 
- gaearon

all are close to 0